### PR TITLE
plane: create basic-setup.rst

### PR DESCRIPTION
**Pull request by vierfuffzig**
content by @Hwurzburg  
thanks a lot henry for providing this!

this is intended as a basis to work on.

### CODE REVIEW

**Comment by Hwurzburg** [comment]
Basti, I had added this to the end of the SERVO section:
**Now check that the servos move in the correct directions to level the plane when moved around while in STAB or FBWA mode. Reverse the sevos with the reverse box for a servo if needed. If you have correctly adjusted the TX channel directions in part B above, then they will move in the correct directions also in MANUAL mode.Now check that the servos move in the correct directions to level the plane when moved around while in STAB or FBWA mode. Reverse the sevos with the reverse box for a servo if needed. If you have correctly adjusted the TX channel directions in part B above, then they will move in the correct directions also in MANUAL mode.**
Also added this in Other Params:
**5. ARMING. Leave all arming params at default. There is no reason to disable these safety checks. You should be able to get a GPS lock even indoors with modern GPS units. Inability to arm due to one of these checks means something has to be corrected.**

**Comment by Hwurzburg** [comment]
Been thinking...maybe should be called Arduplane Quick Setup Guide...
PS: I have had a technical writer review the document and will make some slight non-technical working changes when it gets posted (a lot easier for me to do online rather then in a local repo)
Just need someone to review and merge, we can not hyperlink it into the greater Wiki until I make those edits if the Wiki gate keepers desire...

**Comment by tridge** [comment]
I like the concept, thanks! Feel free to merge when you are happy with it

**Comment by Hwurzburg** [comment]
Thanks, Basti for pulling in the changes.... @tridge I think its ready to merge....it needs a reference somewhere in the main wiki to navigate to it...perhaps in the Plane base TOC?

File: plane/source/docs/basic-setup.rst [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+
+After connecting to MP, verify that your GPS is recognized and you eventually get a satellite fix. Number of sats and
+HDOP are shown in the Flight Data tab of MP. If no gps is detected, you will see a message in the HUD display stating "No GPS". 
+That means you have hardware debug to do!
+
+Possible issues:
+
+1. Connected to wrong serial port (see your hardware docs for which port AP expects) or incorrectly connected (TX/RX swapped).
+2. Serial port settings were changed from default. ArduPlane expects GPS on Serial 3 at 38.4Kbaud per default. 
+3. Though UBLOX and NMEA-type GPS receivers are supported by ArduPilot these days, your GPS receiver is not supported.
```
Comment (WickedShell) [inline plane/source/docs/basic-setup.rst]: We support a lot more then these protocols, do you mean "your GPS receiver may not be supported"

**Comment by rmackay9** [comment]
as discussed this should be linked from this page: http://ardupilot.org/plane/docs/plane-configuration-landing-page.html

File: plane/source/docs/basic-setup.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/basic-setup.rst]: yes, will modify...thanks

**Comment by Hwurzburg** [comment]
Thanks Basti!...moved to my git and #1816 replaces this....

**Pull request closed without merging**